wizards/Policy/PolicyWizard.tsx

```diff
@@ -338,6 +339,31 @@ export function PolicyWizardTemplates(props: { policies: IResource[] }) {
                         validation={isValidKubernetesResourceName}
                     />
 
+                    <WizRadioGroup
+                        path="objectDefinition.spec.pruneObjectBehavior"
+                        label="Prune Object Behavior"
+                        labelHelp="Optional. Specifies how related objects on the managed cluster should be pruned when the policy is removed. The parameter values are None, DeleteIfCreated, and DeleteAll."
```

Comment:
I recommend the following update:

Optional. Specifies how related objects on the managed cluster need to be pruned when the policy is removed. The parameter values are `None`, `DeleteIfCreated`, and `DeleteAll`."

Not sure if you can use monospace, but that is what I would do

Reply:
What do you think of "are" instead of "should be" or "need to be"? As in
> ... how related objects on the managed cluster ***are*** pruned when the policy ...

I didn't think of this originally, but the other options almost make it sound like the user will need to do something

And good idea with the monospace, I will check into that.